packit/api.py

```diff
@@ -216,7 +217,11 @@ def sync_release(
             path = os.path.join(self.dg.local_project.working_dir, "README.packit")
             logger.debug(f"Path of README {path}")
             with open(path, "w") as f:
-                f.write(SYNCING_NOTE)
+                f.write(
+                    SYNCING_NOTE.format(
+                        packit_version=get_distribution("packitos").version
```

Comment:
I have a small suggestion: there is a rare case when this could would produce an exception:
```
DistributionNotFound: The 'packitos' distribution was not found and is required by the application
```
When someone clones the repo, uses packit and does not install it. I recall something like this happened when we used similar code in some other project.

Therefore could we put the get_distribution call into a dedicated function (in utils ideally), wrap it in try/catch, catch DistributionNotFound, and if it's not found, return "NOT-INSTALLED" or something like that?

also please add tests for such function